src/spotlight.cc

```diff
@@ -288,22 +289,61 @@ void Spotlight::onEventDataAvailable(int fd, SubEventConnection& connection)
   } // end while loop
 }
 
+// -------------------------------------------------------------------------------------------------
+void Spotlight::onHIDDataAvailable(int fd, SubHidrawConnection& connection)
+{
+  QByteArray readVal(20, 0);
+  if (::read(fd, static_cast<void *>(readVal.data()), readVal.length()) < 0)
+  {
+    if (errno != EAGAIN)
+    {
+      const bool anyConnectedBefore = anySpotlightDeviceConnected();
+      connection.disable();
+      QTimer::singleShot(0, this, [this, devicePath=connection.path(), anyConnectedBefore](){
+        removeDeviceConnection(devicePath);
+        if (!anySpotlightDeviceConnected() && anyConnectedBefore) {
+          emit anySpotlightDeviceConnectedChanged(false);
+        }
+      });
+    }
+    return;
+  }
+  logInfo(hid) << "Received " << readVal.toHex();
```

Comment:
Should be `logDebug` instead of `logInfo` also more detailed: e.g.:
```suggestion
  logDebug(hid) << "Received " << readVal.toHex() << "from" << connection.path(); 
```

Reply:
The change has been implemented.